test/unit/propMerger.test.js

```diff
@@ -0,0 +1,63 @@
+import { mergeProps } from '../../src/HOFs/propMerger'
+
+describe('mergeProps()', () => {
+  it('should merge the `src` object into the `destination` object', () => {
+    const src = { width: 100, height: 200, imgixParams: {ar: "1:2", dpr: 2} }
+    const destination = { }
+    const result = mergeProps(src, destination);
+
+    expect(result).toEqual(
+      { width: 100, height: 200, imgixParams: { ar: "1:2", dpr: 2 } }
+    )
+  })
+
+  it('should not overwrite destination values with source values', () => {
+    const src = { width: 100, height: 200, imgixParams: {ar: "1:2", dpr: 2} }
+    const destination = { width: 101 }
+    const result = mergeProps(src, destination);
+
+    expect(result).toEqual(
+      { width: 101, height: 200, imgixParams: { ar: "1:2", dpr: 2 } }
+    )
```

Comment:
```suggestion
    const src = { width: 100 }
    const destination = { width: 101 }
    const result = mergeProps(src, destination);

    expect(result).toEqual(
      { width: 101 }
    )
```

Better to exclude data that is not related to the test, the same can be done for the test below